ospf6d/ospf6d.h

```diff
@@ -23,6 +23,7 @@
 
 #include "libospf.h"
 #include "thread.h"
+#include "lib/json.h"
```

Comment:
I don't believe the entirety of ospfv3 needs json.h support.  Please put this in the appropriate files only.

Reply:
resolved